mc2skos/mc2skos.py

```diff
@@ -239,7 +241,13 @@ def main():
         return
 
     supported_formats = ['turtle', 'jskos', 'ndjson']
-    if args.outformat not in supported_formats:
+    if not args.outformat and args.outfile:
+        ext = args.outfile.rpartition('.')[-1]
```

Comment:
Interesting, I didn't know about the `rpartition` method, but it's quite useful here as it returns a 3-tupple even if there's no separator in the string :) There's also `os.path.splitext`